swap/tests/alice_refunds_after_restart_bob_refunded.rs

```diff
@@ -0,0 +1,38 @@
+pub mod harness;
+
+use harness::alice_run_until::is_xmr_lock_transaction_sent;
+use harness::FastCancelConfig;
+use swap::protocol::alice::AliceState;
+use swap::protocol::{alice, bob};
+
+/// Bob locks Btc and Alice locks Xmr. Alice does not act so Bob refunds.
+/// Eventually Alice comes back online and refunds as well.
+#[tokio::test]
+async fn alice_punishes_after_restart_if_bob_already_refunded() {
```

Comment:
Test name is wrong, says `punish` :)

Reply:
wuups thanks! - classic copy-paster. I'll fix it :)